nilearn/input_data/nifti_masker.py

```diff
@@ -423,6 +435,12 @@ def transform_single_imgs(self, imgs, confounds=None, copy=True):
             documentation for details: :func:`nilearn.signal.clean`.
             shape: (number of scans, number of confounds)
 
+        sample_mask : Any type compatible with numpy-array indexing, optional
+            shape: (number of scans - number of volumes removed, )
+            Masks the niimgs along time/fourth dimension to perform scrubbing
+            (remove volumes with high motion) and/or non-steady-state volumes.
+            This parameter is passed to signal.clean.
```

Comment:
I think we could add a `versionadded` directive (for all maskers) to advertise for this new feature. WDYT?

```python
sample_mask : type
    Description...

        .. versionadded:: 0.7.2

```

Reply:
same